### PR TITLE
Fix setting language link to USER/visible during ingest

### PR DESCRIPTION
**Pull request by joel-wright**
# Description

This PR updates the ingest code to switch language links supplied in an ingested CSV to "supplied by the user & should be visible" when appropriate.

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Additional test to validate the change.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -45,6 +45,7 @@ pyright = "^1.1.294"
 pytest-asyncio = "^0.18.3"
 ruff = "^0.0.246"
 pytest-alembic = "^0.10.5"
+types-SQLAlchemy = "^1.4.31"
```
Comment (THOR300) [inline pyproject.toml:48]: Where is this used?

Comment (THOR300) [inline pyproject.toml:48]: Where is this used?

**Review by THOR300 — APPROVED** [review]
LGTM just thinking whether we need to add a test that mimics the db writer update and then assert that the document language that's added is not visible.

**Comment by joel-wright** [comment]
> LGTM just thinking whether we need to add a test that mimics the db writer update and then assert that the document language that's added is not visible.

I think this is already covered by the `test_document_families.py` tests - there are tests there that assert that the languages are added with a source of `LanguageSource.MODEL`... is that what you meant?

**Pull request merged**